os/fs/smartfs/smartfs.h

```diff
@@ -221,8 +221,10 @@
 
 /* Buffer flags (when CRC enabled) */
 
-#define SMARTFS_BFLAG_DIRTY       0x01	/* Set if data changed in the sector */
-#define SMARTFS_BFLAG_NEWALLOC    0x02	/* Set if sector not written since alloc */
+#define SMARTFS_BFLAG_UNMOD		0x00	/* Indicates that the buffer in unmodified, no changes to be recorded in MTD */
```

Comment:
Usually use 'ideal' 'clean' 'empty' or something has similar meaning with this, but It is first time that I saw kind of this name.
Is this better than all of these?

Reply:
UNMOD -> Un Modified
Since we are recording changes in the sf buffer to MTD
Buffer flags set as Un Modified means no changes in buffer to record to MTD.
I think this name is very apt.

Reply:
got it. but description need to be changed, 'Set if ....(To represent initial state & can be set if we don't want write this entry....blahblah)

Reply:
I think the description is alright too.
Same as the above comment...

Reply:
Ohkkay I can align it with other descriptions: -
"Set if ......"